docs/conf/portland/2025/opportunity-grants.md

```diff
@@ -18,7 +18,7 @@ You are welcome to apply, even if you have received a grant before from our conf
 
 We prioritize applications based on the overall impact that granting an application will have on the applicant, the Write the Docs community, and the applicant’s wider community and country. However, there are no requirements for you to meet. In other words, if you’re unsure, please apply.
 
-Grant applicants, like all other participants in the Write the Documents community, are required to follow the [Code of Conduct](https://www.writethedocs.org/conf/portland/2025/code-of-conduct/).
+Grant applicants, like all other participants in the Write the Docs community, are required to follow the [Code of Conduct](https://www.writethedocs.org/conf/portland/2025/code-of-conduct/).
```

Comment:
I don't seem to have the ability to do a suggestion here, but while reviewing your (excellent) changes, I noticed that in the phrase "Write the Docs community" that the "Write the Docs" part of it is a compound adjective, and so should be hyphenated unless I am missing something:  thus  "participants in the Write-the-Docs community"